ethereum/eth/src/test/java/org/hyperledger/besu/ethereum/eth/sync/RangeHeadersValidationStepTest.java

```diff
@@ -76,6 +74,7 @@ public void setUp() {
 
   @Test
   public void shouldValidateFirstHeaderAgainstRangeStartHeader() {
+    setUp();
```

Comment:
Seems a bit odd this one, why not use `@BeforeEach`?

Reply:
there's one test that doesn't need the setup. It was either this or create a separate test class without the setup. I went back and forth